### ISSUE TITLE
Introduce soil and seeds clusters management

### ISSUE THREAD

**Issue by kayrus** [gardener/garden-setup#48]
**What would you like to be added**:

acre.yaml has to have an ability to create soil and underlying seeds clusters.

**Why is this needed**:

simplify the overall cascade gardener installation

**Comment by kayrus** [comment]
@Diaphteiros since `landscape.iaas` was refactored, can we create the seed tree? e.g.

```yaml
iaas:
  - name: (( type ))                           # name of the seed
    type: <gcp|aws|azure|openstack>            # iaas provider
    region: <major region>-<minor region>      # region for initial seed
    zones:                                     # remove zones block for Azure
      - <major region>-<minor region>-<zone>   # example: europe-west1-b
      - <major region>-<minor region>-<zone>   # example: europe-west1-c
      - <major region>-<minor region>-<zone>   # example: europe-west1-d
    credentials: ...
    seeds: # <<<<<<<<-- here
    - name: (( type ))                           # name of the seed
      type: <gcp|aws|azure|openstack>            # iaas provider
      region: <major region>-<minor region>      # region for initial seed
      zones:                                     # remove zones block for Azure
        - <major region>-<minor region>-<zone>   # example: europe-west1-b
        - <major region>-<minor region>-<zone>   # example: europe-west1-c
        - <major region>-<minor region>-<zone>   # example: europe-west1-d
      credentials:
```

**Comment by Diaphteiros** [comment]
Why do you want a nested seed tree? What is your intention behind that?

**Comment by kayrus** [comment]
@Diaphteiros simplify and unify the automation to build soil + regional seed clusters.

**Comment by Diaphteiros** [comment]
I was thinking about a flat structure, but I can see some advantages in your proposal. I'll think about it, but I'll probably follow your suggestion.

**Comment by Diaphteiros** [comment]
fixed with #67 and documented [here](https://github.com/gardener/garden-setup/blob/master/docs/extended/iaas.md#shooted-seeds)
